code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm

```diff
@@ -3,24 +3,25 @@
 
 ////////////////////////////////////////////////DONUTS////////////////////////////////////////////////
 
-/datum/crafting_recipe/food/chaosdonut
-	name = "Chaos donut"
+/datum/crafting_recipe/food/donut
+	time = 15
+	name = "Donut"
 	reqs = list(
-		/datum/reagent/consumable/frostoil = 5,
-		/datum/reagent/consumable/capsaicin = 5,
+		/datum/reagent/consumable/sugar = 1,
 		/obj/item/reagent_containers/food/snacks/pastrybase = 1
 	)
-	result = /obj/item/reagent_containers/food/snacks/donut/chaos
+	result = /obj/item/reagent_containers/food/snacks/donut/plain
 	subcategory = CAT_PASTRY
 
-/datum/crafting_recipe/food/donut
-	time = 15
-	name = "Donut"
+
+/datum/crafting_recipe/food/donut/chaos
+	name = "Chaos donut"
 	reqs = list(
-		/datum/reagent/consumable/sugar = 1,
+		/datum/reagent/consumable/frostoil = 5,
+		/datum/reagent/consumable/capsaicin = 5,
 		/obj/item/reagent_containers/food/snacks/pastrybase = 1
 	)
-	result = /obj/item/reagent_containers/food/snacks/donut
+	result = /obj/item/reagent_containers/food/snacks/donut/chaos
 	subcategory = CAT_PASTRY
```

Comment:
/datum/crafting_recipe/food/donut already has `subcategory` set to CAT_PASTRY, no need to define it twice

Same goes for all the other child datums like /datum/crafting_recipe/food/donut/slimejelly